raiden/network/proxies/token_network.py

```diff
@@ -2255,6 +2255,66 @@ def settle(
 
         log.info("settle successful", **log_details)
 
+    def settle_light(
```

Comment:
Couldn't settle and settle_light be generalized? They have the same base structure.

Reply:
yes. Anyway, it's true it may be a bit too much time expensive right now.

Reply:
Not at all, i will try to simplify this code and make it more general